dev-db/pgcli/pgcli-3.0.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
`dev-python/pgspecial` needs porting to 3.8 and 3.9 before pgcli (should be handled in https://github.com/gentoo/gentoo/pull/17405)

Reply:
Added 3.8 and 3.9